android/src/main/java/com/google/android/cameraview/Camera1.java

```diff
@@ -829,6 +829,8 @@ public void onAutoFocus(boolean success, Camera camera) {
                     }
                 });
             }
+
+            setExposureInternal(-1);
```

Comment:
why -1?

Reply:
It could be any value <=0 or >1. It's just -1, because the default value is also -1.

Reply:
exposure on iOS is a float, which can't be nil. B/c the range-value is between 0 and 1 we need a value to indicate that exposure isn't set, which in this case is -1. It's -1 on Android to keep the implementation consistent across platforms.